lib/manageiq/loggers/journald.rb

```diff
@@ -1,8 +1,10 @@
 module ManageIQ
   module Loggers
     class Journald < Base
-      # An syslog identifier used when writing messages. The default is the progname.
-      attr_accessor :syslog_identifier
+      require "active_support/core_ext/module/aliasing"
+
+      # Alias syslog_identifier for backwards compatibility
+      alias_attribute :syslog_identifier, :progname
```

Comment:
This is probably overkill, looks like we don't use this functionality currently (https://github.com/ManageIQ/manageiq-loggers/search?q=syslog_identifier) so we can probably drop this without _really_ breaking compatibility

Reply:
I think master will be a major release, so we might not need it, but I'm fine with it for now.

Reply:
I'm fine with dropping it if we're cutting a major release

Reply:
:+1: removed it